index.js

```diff
@@ -600,4 +600,7 @@ function fastifyView (fastify, opts, next) {
   }
 }
 
-module.exports = fp(fastifyView, { fastify: '>=3.x' })
+module.exports = fp(fastifyView, {
+  fastify: '>=3.x',
```

Comment:
```suggestion
  fastify: '3.x',
```

This is more accurate now that we are working on this.

Reply:
fastify version [updated](https://github.com/fastify/point-of-view/pull/199/commits/a6acf2b3a99daa09ec37a61425d822455fb19757)